src/utils/generateFlowDiagramData.js

```diff
@@ -0,0 +1,88 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import {
+  ADMISSIONS,
+  ADMISSIONS_NEW_COURT,
+  ADMISSIONS_REVOCATIONS_PAROLE,
+  ADMISSIONS_REVOCATIONS_PROBATION,
+  metricToCardName,
+  POPULATION_PAROLE,
+  POPULATION_PRISON,
+  POPULATION_PROBATION,
+  RELEASES,
+} from "../constants/metrics";
+import generateHint from "./generateHint";
+import months from "../constants/months";
+
+const generateFlowDiagramData = (data) => {
```

Comment:
Let's add a docstring explaining what this is doing, what the inputs are, and what the outputs are.